components/PrimaryButton/PrimaryButton.styled.js

```diff
@@ -0,0 +1,23 @@
+import styled from "styled-components";
+
+export const PrimaryButton = styled.button`
```

Comment:
Nur meine Meinung: 
Ich glaube, dass du dir das Leben mit styled-components schwer machst. Du benutzt es hier wie CSS, also könntest du auch einfach pures CSS z. B. als CSS modules benutzen. styled-components scheint, wenn du ein Theme konfiguriert hast oder dynamische Styles basierend auf props brauchst. Aber auch dafür ist styled-components nicht mehr die beste Wahl.

Reply:
I will add a `lessons learned`section in the README.md at the end of the project. I will write about the learnings according to the architecture I used for styling the elements. So that's a good point to write about.